hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-applicationhistoryservice/src/main/java/org/apache/hadoop/yarn/server/timeline/LeveldbTimelineStore.java

```diff
@@ -242,19 +240,7 @@ protected void serviceInit(Configuration conf) throws Exception {
       IOUtils.cleanupWithLogger(LOG, localFS);
     }
     LOG.info("Using leveldb path " + dbPath);
-    try {
-      db = factory.open(new File(dbPath.toString()), options);
-    } catch (IOException ioe) {
-      File dbFile = new File(dbPath.toString());
-      File backupPath = new File(
-          dbPath.toString() + BACKUP_EXT + Time.monotonicNow());
-      LOG.warn("Incurred exception while loading LevelDb database. Backing " +
-          "up at "+ backupPath, ioe);
-      FileUtils.copyDirectory(dbFile, backupPath);
-      LOG.warn("Going to try repair");
```

Comment:
Would you keep the logging message in the helper method? Thanks

Reply:
I think we can remove that. Thanks for the suggestion.

Reply:
I just checked it's already removed from the helper method.

Reply:
Sorry my previous comment was confusing. I want to add the logging messages. I think the messages are useful for the administrators.

Reply:
Added the logging message in the helper method.